Gemfile

```diff
@@ -53,7 +56,8 @@ gem 'iiif_manifest', '~> 0.6'
 gem 'rack-cors', require: 'rack/cors'
 gem 'rails_same_site_cookie'
 gem 'recaptcha', require: 'recaptcha/rails'
-gem 'samvera-persona', '~> 0.3'
+#gem 'samvera-persona', '~> 0.3'
+gem 'samvera-persona', git: "https://github.com/samvera-labs/samvera-persona.git", branch: "newer_rails"
```

Comment:
Temporarily use branch until PR is merged and release is cut.  This PR doesn't need to wait on this though since we can clean that up in a future PR before next avalon release.